ament_copyright/ament_copyright/main.py

```diff
@@ -124,8 +124,8 @@ def main(argv=sys.argv[1:]):
             parser.error(
                 "'LICENSE' argument must be a known license name. "
                 "Use the '--list-licenses' options to see alist of valid license names.")
-        license = licenses[args.add_missing[1]]
-        add_missing_header(file_descriptors, name, license, args.verbose)
+        add_missing_header(
+            file_descriptors, name, licenses[args.add_missing[1]], args.verbose)
```

Comment:
what was the error we're addressing here? if it's that we don't want 'wasteful' variables I don't think we should enforce that across the whole codebase as a general rule, because it's usually done for improved readability. but it may have been for another reason.

Reply:
`license` is a built-in in Python and should not be used as a variable name.